packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -50,6 +50,16 @@ export const globalFlags = {
 // eslint-disable-next-line @rushstack/typedef-var
 export const globalOptions = {
   // Account
+  accountAlias: createOption({
+    key: 'accountAlias' as const,
+    defaultIsOptional: false,
+    prompt: account.accountAliasPrompt,
+    validation: z.string(),
+    option: new Option(
+      '-a, --account-alias <accountAlias>',
+      'Enter an alias to store your account',
+    ),
+  }),
```

Comment:
I wouldn't use the `-a` flag for `accountAlias` and reserve it for `account`

Reply:
As part of cleaning up global options we thought of look into this deeper because currently few other options also use `-a` at the moment.